doc/usage/configuration.rst

```diff
@@ -2087,6 +2082,33 @@ These options influence LaTeX output.
       This overrides the files which is provided from Sphinx such as
       ``sphinx.sty``.
 
+.. confval:: latex_theme
+
+   The "theme" that the LaTeX output should use.  It is a collection of settings
+   for LaTeX output (ex. document class, top level sectioning unit and so on).
+
+   As a built-in LaTeX themes, ``manual`` and ``howto`` are bundled.
+
+   ``manual``
+     A LaTeX theme for writing a manual.  It based on ``report`` document class
+     (``jsbook`` is also used for Japanese documents).
+
+   ``howto``
+     A LaTeX theme for writing an article.  It based on ``article`` document class
+     (``jreport`` is also used for Japanese documents).  :confval:`latex_appendices`
```

Comment:
It imports the ``article`` document class (Japanese documents use ``jreport`` rather).